streaming_data_types/array_1d_se00.py

```diff
@@ -107,8 +107,8 @@ def deserialise_se00(buffer: Union[bytearray, bytes]) -> Response:
 
     max_time = datetime(
         year=9000, month=1, day=1, hour=0, minute=0, second=0
-    ).timestamp()
-    used_timestamp = SE_data.PacketTimestamp() / 1e9
+    ).timestamp() * 1e9
+    used_timestamp = SE_data.PacketTimestamp()
     if used_timestamp > max_time:
         used_timestamp = max_time
```

Comment:
Any idea what this is for?

Reply:
I multiplied the max time (year 9000) with 1e9 to get it nanoseconds, and to keep everything in nanoseconds I removed the /1e9 for the used_timestamp

Reply:
That's not what I meant: why is there a check against some random time in the year 9000?